hq/app/aws/ec2/EC2.scala

```diff
@@ -56,7 +56,9 @@ object EC2 {
       sgResult <- TrustedAdvisorSGOpenPorts.getSGOpenPorts(supportClient)
       sgUsage <- getSgsUsage(sgResult, account)
       flaggedSgs = sgResult.flaggedResources.filter(_.status != "ok")
-    } yield sortSecurityGroupsByInUse(flaggedSgs, sgUsage)
+      vpcs <- getVpcs(account, flaggedSgs)(getVpcsDetails)
```

Comment:
It looks to me like if we fail to fetch the VPC information this whole feature will fail. While that may be what we want, it's also possible that since this is just "extra" information we'll be ok to log the errors and carry on?

Have a think about whether you really mean for this to be required. If not, you could use something like `Attempt.successfulAttempts` to discard the failures (obviously making sure they're at least logged and we don't discard them completely).

On the other hand, that would raise the question of "what do we do about the failure" and "who's ever going to look at the logs", so I'm happy if the answer is "it's fine like this".

Reply:
I cannot see how retrieving vpc details could fail, except a technical failure, which is a different matter. VPC details are retrieved like any other AWS api call. I think it feels like too cautious to add this safety guard.

Reply:
OK!